content/docs/dvclive/api-reference/live/log.md

```diff
@@ -19,10 +16,16 @@ live.log("loss", 0.9)
 
 ## Description
 
-The first call to `live.log(name, val)` will create a new file in
-`{path}/{name}.tsv` including the header and first row.
+If `summary` is True, `live.log(name, val)` will update the
+[summary](/doc/dvclive/get-started#summary) with the latest value logged.
+
+💡 The summary `{path}.json` is usable by `dvc metrics`.
+
+### Step updates
```

Comment:
Overall, the `step` info seems a bit scattered. Maybe we can move this section into get started, possibly make it more high level, and then reference to it throughout the docs when discussing steps (like `summary` and `history`).

Reply:
I kind of have gone in the opposite direction in #3207 because the implications of updating `step` vary across the type of data being logged.

I removed `summary` / `history` info from `Outputs` section in favor of:


> The contents of the dvclive folder and dvclive.json would vary depending on the type of data you have logged and whether you have updated the step value or not. 
> See Live.log(), Live.log_image() and Live.log_plot() for more details.

Reply:
Sounds good, we can revisit in that PR.